hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.h

```diff
@@ -55,13 +43,12 @@ namespace hazelcast {
 
                 struct HAZELCAST_API client_addclusterviewlistener_handler : public impl::BaseEventHandler {
                     void handle(ClientMessage &msg);
-
                     /**
                      * @param version Incremental member list version
-                     * @param memberInfos List of member infos  at the cluster associated with the given version
-                     *                    params:
+                     * @param member_infos List of member infos  at the cluster associated with the given version
+                     *                     params:
                     */
-                    virtual void handle_membersview(int32_t version, std::vector<member> const &member_infos) = 0;
+                    virtual void handle_membersview(int32_t version, std::vector<member> const & member_infos) = 0;
```

Comment:
extra space after `&`

Reply:
These existed in the protocol repo and was deleted manually here. They came back since I regenerated. I fixed this in the protocol repo (see hazelcast/hazelcast-client-protocol@3a0bf96) and now it's fine.